src/routes/+page.server.ts

```diff
@@ -35,7 +35,7 @@ export const actions: Actions = {
                 "title": `${crypto.randomUUID().toString()}`,
                 "description": `${crypto.randomUUID().toString()}`
             }, token);
-            event.cookies.set("banner_id", bannerCreate.id, { path: '/', sameSite: 'strict' });
+            event.cookies.set("banner_id", bannerCreate.id, { path: '/', sameSite: 'strict', httpOnly: false });
```

Comment:
We should consider setting httpOnly back to true.
The cookies without httpOnly flag are vulnerable to attacks, and the token may be leaked as it is accessible by using JS in the browser.